Blog Post | Considerations for A Digital Nomad's First Trip (#52)
shipshapecode/website-astro: data/posts/2022-11-04-considerations-before-first-nomad-trip.md

### Time Zones

The time zone of the location you are working in is a huge consideration. Some nomads prefer working in the same or very similar time zones to where their other colleagues are based. Others don't mind working further and making moderate to relatively large adjustments in their sleep cycle and structure of their day. It’s often a good idea to consider working in areas that will allow you to overlap for at least a few hours per day so that you will have some time to collaborate with your colleagues in real-time when needed. 

## Lifestyle

> I would recommend giving some thought to what kind of lifestyle you'll want to live while abroad. In order to do so, at a minimum, you'll need to consider the cost of living of places you are interested in, preferences for your accomodation and working, ease of access to service and goods, and logistics for traveling domestically or otherwise in destinations of interest to you. 

Reviewer: ```suggestion
I would recommend giving some thought to what kind of lifestyle you'll want to live while abroad. In order to do so, at a minimum, you'll need to consider the cost of living of places you are interested in, preferences for your accommodation and working, ease of access to services and goods, and logistics for traveling domestically or otherwise in destinations of interest to you. 
```

### Cost of Living

 A major factor in how you'll live while abroad is the cost of living. When I plan a trip, a lower cost of living is often a draw for me. Although I don't entirely avoid higher cost of living locations, I definitely stay there for less time than I do less expensive locations. When planning a trip you should take a look at the cost of living for the specific cities or towns you're curious about and pay attention to average prices for food and lodging in particular. 

> ### Your Accomodation and Workplace

Reviewer: ```suggestion
### Your Accommodation and Workplace
```

> I would definitely recommend finding out where the more central neighborhoods are where you're staying and trying to stay in one of those if they are considered to be safe. This generally costs a bit more, but allows you to make more of the time you have there. Staying somewhere that is a bit further out, but well connected by public transit is also something to consider, especially if you're going to be staying awhile. I really prefer to rent an apartment and often work from there, but I don't think everyone should necessarily do that -- many nomads really enjoy both living in and working in shared accomodations.

Reviewer: ```suggestion
I would definitely recommend finding out where the more central neighborhoods are where you're staying and trying to stay in one of those if they are considered to be safe. This generally costs a bit more, but allows you to make more of the time you have there. Staying somewhere that is a bit further out, but well connected by public transit is also something to consider, especially if you're going to be staying awhile. I really prefer to rent an apartment and often work from there, but I don't think everyone should necessarily do that -- many nomads really enjoy both living in and working in shared accommodations.
```

> Some hostels or more communal accomodations do have quieter common areas with desk space where you could work if that's an option that is interesting to you. As an example, I once stayed at Cityhub in Copenhagen and was able to get work done in the shared area near the kitchen with no problem. Additionally, if you have a different working preference than me and really prefer to get out of the house, you can try to look into local options such as cafes. If you think you may want to often do work at cafes, try to find out ahead of time if outlets are available at the cafes you are interested in. Another option, especially common in digital nomad hubs, is to work at local co-working places. This will allow you to have space to work and also meet other people who are working while traveling as well

Reviewer: ```suggestion
Some hostels or more communal accommodations do have quieter common areas with desk space where you could work if that's an option that is interesting to you. As an example, I once stayed at Cityhub in Copenhagen and was able to get work done in the shared area near the kitchen with no problem. Additionally, if you have a different working preference than me and really prefer to get out of the house, you can try to look into local options such as cafes. If you think you may want to often do work at cafes, try to find out ahead of time if outlets are available at the cafes you are interested in. Another option, especially common in digital nomad hubs, is to work at local co-working places. This will allow you to have space to work and also meet other people who are working while traveling as well
```

When leaving a country, you may need to show your visa, so make sure you keep it in a safe place. If you want to travel to other countries, you might also want to take a look at those entry requirements as well. For instance, if you were visiting Thailand and interested in visiting some neighboring countries like Vietnam, getting a visa for Vietnam might require that you have additional passport photos -- you may want to have those printed ahead of time. Another thing to keep in mind is that some countries require proof of an onward ticket -- if you don't have one, airlines may refuse to allow you to board, so check into this in order to make sure you are prepared before arriving at the airport and don't have to hastily book a ticket.

## What to Bring

> This varies quite a bit. I am not a carry-on only packer -- for me, the stress versus thebenefits aren't really there. Lighter is better, but you do want to make sure you have the stuff you need. Often, you don't need to bring a huge amount of clothes, but if the clothing or shoe sizes you wear are not easy to find in the locations you are considering visiting, make sure to bring enough of those items in case you can't obtain them with ease locally. Make sure to bring medications ahead of time -- I would recommend bringing some sort of pain relief pills as well as digestive medicines so you have them ahead of time -- and fill any necessary prescriptions before you leave. In addition to your computer, phone, and charger, you may want to consider a set of noise-canceling headphones, and I would definitely suggest you bring some external battery packs that could be used to charge your devices on the go.

Reviewer 2: ```suggestion
This varies quite a bit. I am not a carry-on only packer -- for me, the stress versus the benefits aren't really there. Lighter is better, but you do want to make sure you have the stuff you need. Often, you don't need to bring a huge amount of clothes, but if the clothing or shoe sizes you wear are not easy to find in the locations you are considering visiting, make sure to bring enough of those items in case you can't obtain them with ease locally. Make sure to bring medications ahead of time -- I would recommend bringing some sort of pain relief pills as well as digestive medicines so you have them ahead of time -- and fill any necessary prescriptions before you leave. In addition to your computer, phone, and charger, you may want to consider a set of noise-canceling headphones, and I would definitely suggest you bring some external battery packs that could be used to charge your devices on the go.
```

> Additionally, before arriving in the country you may want to do some preparation and download apps or information to make your life easier, such as offline maps, translations for languages that are spoken in the locations you'll been traveling to, and apps for the local rideshare services.

Reviewer: ```suggestion
Additionally, before arriving in the country you may want to do some preparation and download apps or information to make your life easier, such as offline maps, translations for languages that are spoken in the locations you'll be traveling to, and apps for the local rideshare services.
```
